### PR TITLE
Fix In the lab video links

### LINKED ISSUE #24: "In the Lab" video player should play only one video at a time
You can start multiple videos playing at once.

Steps to reproduce:
1. Start an experiment with micrsoscopy
2. Advance to the "Select Technique" tab
3. In the "In the Lab" video player (middle right) press the play button. This should start a video.
4. Advance to the next button using the right-pointing green arrow. This should bring up the YouTube thumbnail from a second video. 
5. Press Play button

expected result: 1st video stops, 2nd video starts
actual result: both videos play at the same time

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
Fix #24
#### What's this PR do?
Change "In the Lab" embedded videos to be link images to youtube.
Now when you click on the image a new tab opens up with the corresponding video on youtube.

### CODE REVIEW

**Comment by pdpinch** [comment]
Looks like the thumbnail images are missing?

Otherwise, I think it's functioning the way you want it too. And it won't play two videos at the same time the way it used to.

File: html_app/ui/facs.soy [hunk 1]
```diff
@@ -516,16 +516,16 @@ FACS Main View
...
+                    </a>
                 </div>
                 <div>
-                    <iframe height="124" width="187" controls="0"
-                        class='scb_s_experiment_setup_video_box_vid'
-                        src="https://www.youtube.com/embed/W9p6Y8IW89A" frameborder="0">
-                    </iframe>
+                    <a href="//youtu.be/W9p6Y8IW89A-Y" target="blank">
+                        <img height='124' width='187'
+                        src="images/microscopy/videos/video_3.jpg">
```
Comment (pdpinch) [inline html_app/ui/facs.soy]: Shouldn't this be in the facs directory?

File: html_app/ui/western_blot.soy [hunk 2]
```diff
@@ -573,28 +573,28 @@
                     <span class='scb_s_western_blot_video_text_second'>lane must be reserved for the protein marker.&nbsp;</span><br/>
                 </div>
                 <div>
-                    <iframe height='124' id="wb_video1" width='187'
-                        class='scb_s_experiment_setup_video_box_vid'
-                        src="https://www.youtube.com/embed/GJJGNOdhP8w" frameborder="0">
-                    </iframe>
+                    <a href="//youtu.be/GJJGNOdhP8w-Y" target="blank">
+                        <img height='124' width='187'
+                        src="images/western_blot/videos/video_1.jpg">
```
Comment (pdpinch) [inline html_app/ui/western_blot.soy]: should this be `images/western_blot/youtube_links/video_1.jpg` ?

File: html_app/ui/western_blot.soy [hunk 3]
```diff
@@ -573,28 +573,28 @@
...
+                    </a>
                 </div>
                 <div>
-                    <iframe height='124' id="wb_video_2" width='187'
-                        class='scb_s_experiment_setup_video_box_vid'
-                        src="https://www.youtube.com/embed/JcN0EkcHrKk" frameborder="0">
-                    </iframe>
+                    <a href="//youtu.be/JcN0EkcHrKk-Y" target="blank">
+                        <img height='124' width='187'
+                        src="images/western_blot/videos/video_2.jpg">
```
Comment (pdpinch) [inline html_app/ui/western_blot.soy]: should this be `images/western_blot/youtube_links/video_2.jpg` ?

File: html_app/ui/western_blot.soy [hunk 4]
```diff
@@ -573,28 +573,28 @@
...
+                    </a>
                 </div>
                 <div>
-                    <iframe height='124' id="wb_video_3" width='187'
-                        class='scb_s_experiment_setup_video_box_vid'
-                        src="https://www.youtube.com/embed/IoVzpL_heFo" frameborder="0">
-                    </iframe>
+                    <a href="//youtu.be/IoVzpL_heFo-Y" target="blank">
+                        <img height='124' width='187'
+                        src="images/western_blot/videos/video_3.jpg">
```
Comment (pdpinch) [inline html_app/ui/western_blot.soy]: should this be `images/western_blot/youtube_links/video_3.jpg` ?

File: html_app/ui/western_blot.soy [hunk 5]
```diff
@@ -573,28 +573,28 @@
...
+                    </a>
                 </div>
                 <div>
-                    <iframe height='124' id="wb_video_4" width='187'
-                        class='scb_s_experiment_setup_video_box_vid'
-                        src="//www.youtube.com/embed/u7VwmJw9Gbc" frameborder="0">
-                    </iframe>
+                    <a href="//youtu.be/u7VwmJw9Gbc-Y" target="blank">
+                        <img height='124' width='187'
+                        src="images/western_blot/videos/video_4.jpg">
```
Comment (pdpinch) [inline html_app/ui/western_blot.soy]: should this be `images/western_blot/youtube_links/video_4.jpg` ?

File: html_app/ui/microscopy.soy [hunk 6]
```diff
@@ -915,22 +915,22 @@
...
+                </a>
             </div>
             <div>
-                <iframe height='124' width='187' controls='0'
-                    class='scb_s_experiment_setup_video_box_vid'
-                    src="//www.youtube.com/embed/W9p6Y8IW89A" frameborder="0">
-                </iframe>
+                <a href="//youtu.be/W9p6Y8IW89A-Y" target="blank">
+                    <img height='124' width='187'
+                    src="images/microscopy/videos/video_3.jpg">
```
Comment (pdpinch) [inline html_app/ui/microscopy.soy]: I think these images have the wrong path too.

**Review by pdpinch — CHANGES_REQUESTED** [review]
I see the images, but It looks like the paths are wrong, or the images aren't in the right directories.

File: html_app/ui/facs.soy (hunk 1 above)
Comment (annagav) [reply html_app/ui/facs.soy]: FACS and Microscopy have one overlapping video.

**Comment by annagav** [comment]
@pdpinch I think I fixed all the links.

**Pull request merged**